docs/how-to/configure-revision-history.md

```diff
@@ -0,0 +1,25 @@
+# How to configure revision history
+
+You can limit how many versions of the images are kept in OpenStack glance by specifying the
+revision-history-limit configuration option.
+
+```
+REVISION_HISTORY_LIMIT=2
+OPENSTACK_AUTH_URL=<openstack-auth-url, e.g. http://my-openstack-deployment/openstack-keystone>
+OPENSTACK_PASSWORD=<openstack project password>
+OPENSTACK_PROJECT_DOMAIN_NAME=<openstack project domain name>
+OPENSTACK_PROJECT_NAME=<openstack project name>
+OPENSTACK_USER_DOMAIN_NAME=<openstack user domain name>
+OPENSTACK_USERNAME=<openstack username>
+
+juju deploy github-runner-image-builder \
+--config revision-history-limit=$REVISION_HISTORY_LIMIT
+--config openstack-auth-url=$OPENSTACK_AUTH_URL \
+--config openstack-password=$OPENSTACK_PASSWORD \
+--config openstack-project-domain-name=$OPENSTACK_PROJECT_DOMAIN_NAME \
+--config openstack-project-name=$OPENSTACK_PROJECT_NAME \
+--config openstack-user-domain-name=$OPENSTACK_USER_DOMAIN_NAME \
+--config openstack-user-name=$OPENSTACK_USERNAME
+```
+
+The example above would keep two revisions of the image before deletion.
```

Comment:
s/two revisions of the image before deletion/ the two most recent revisions of the image